docs/api-reference/layers/editable-geojson-layer.md

```diff
@@ -354,3 +354,29 @@ The pointer moved, regardless of whether the pointer is down, up, and whether or
 * `groundCoords` (Array): `[lng, lat]` ground coordinates where the pointer is now.
 * `isDragging` (Boolean): `true` if the pointer is moving but it is considered a drag, in this case you likely want to handle `onDragging`
 * `pointerDownPicks` (Array): An array containing [deck.gl Picking Info Objects](https://uber.github.io/deck.gl/#/documentation/developer-guide/adding-interactivity?section=what-can-be-picked-) for all objects that were under the pointer when it went down, or `null` if pointer is moving without pointer down.
+
+
+## Define Sublayers
+
+Three default sublayers make up the composite `EditableGeoJsonLayer`:
+
+1. `GeometryLayer` renders the geometries. Default: `GeoJsonLayer`
+2. `PointHandleLayer` renders the point edit handles. Default: `ScatterplotLayer`
+3. `IconHandleLayer` renders the icon edit handles. Default `IconLayer`
```

Comment:
Shouldn't these 2 be the same? Is it possible to use both at the same time?

Reply:
Agreed here. I think if we are making this flexible enough to pass in your own layer, `IconHandleLayer` doesn't need it's own props. It can just be in the deck example to show the flexibility of this prop. Then instead of `PointHandleLayer` and `IconHandleLayer`, it'd just be `EditHandleLayer` with the default value being `ScatterplotLayer`.

Reply:
My thought here was that these objects serve more to extend the existing props of the default sublayers. But I see why this and the `IconLayer` props are redundant, so I'll change it.

Should it be the case that these layers exclusively use the sublayer-specific props that are defined for them? Or should they overwrite/add to the existing proxy props like in the example above?